README.md

```diff
@@ -37,7 +37,16 @@ const unauthorizationCallback = () => {
   // Implement redirection to login page here
 };
 
+const useAuth: boolean = // Optional: default is true, use false if you want to disable auth;
+
+// in case of using auth
 ApiKitClient.initialize(BASE_URL, authTokenCallback, unauthorizationCallback);
+
+// Or
+ApiKitClient.initialize(BASE_URL, authTokenCallback, unauthorizationCallback, true);
+
+// in case of not using auth
+ApiKitClient.initialize(BASE_URL, authTokenCallback, unauthorizationCallback, false);
 ```
```

Comment:
There is inconsistency in the code and README.md

1. In code the default is `false`. here is it `true`

2. and in case of not wanted Auth it should be  as below right? 👀 
// in case of not using auth
ApiKitClient.initialize(BASE_URL);

Reply:
Yes, thanks again, not revisit this as well! 😄 🙏

Reply:
Done ✅